test-utils-core/src/test/java/uk/gov/justice/services/test/utils/core/matchers/ResponsePayloadMatcherTest.java

```diff
@@ -36,7 +36,7 @@ public void shouldMatchJsonPayloadFromResponse() throws Exception {
 
         final String eventsJson = createObjectBuilder().add("events", events).build().toString();
 
-        assertThat(new ResponseData(null, eventsJson), payload()
+        assertThat(new ResponseData(null, eventsJson, null), payload()
```

Comment:
Is this going to change all existing tests to need to add a null param if they don't want to test headers (which they won't be). Could we add a different constructor that defaults to null header?

- Apologies if I misunderstand what this is doing.